core/src/main/java/gov/nist/secauto/metaschema/core/metapath/function/library/FnCompare.java

```diff
@@ -41,7 +41,11 @@
 
 import edu.umd.cs.findbugs.annotations.NonNull;
 
-public class FnCompare {
+public final class FnCompare {
```

Comment:
Minor, non-blocking: should we add a Javadoc string to match other functions we are adding?

```suggestion
/**
 * Implements <a href=
 * "https://www.w3.org/TR/xpath-functions-31/#func-compare">fn:compare</a>.
 */
public final class FnCompare {
```